index.js

```diff
@@ -607,9 +607,8 @@ WuiDom.prototype.delClassNames = function (classNames) {
  * @param {Boolean} [shouldAdd]
  */
 WuiDom.prototype.toggleClassNames = function (classNames, shouldAdd) {
-	classNames = joinArgumentsAsClassNames('', classNames);
-
-	if (arguments.length > 1) {
+	if (shouldAdd !== undefined) {
+		classNames = joinArgumentsAsClassNames('', classNames);
```

Comment:
I certainly wouldn't mind being able to pass in just a string. Usually I'm only checking against one classname anyway. Then we could only perform a join if we receive an array to begin with, and only if we're adding the classname.